examples/skip_plot_map_in_3d.py

```diff
@@ -39,7 +38,10 @@
 plotter.plot_line(line)
 
 # Define a SkyCoord for to set the positon of the camera
-camera_position = SkyCoord(0*u.deg, 0*u.deg, 8*const.R_sun, obstime=m.observer_coordinate.obstime, frame=frames.HeliographicStonyhurst)
-plotter.set_camera_coordinates(camera_position)
+camera_position = SkyCoord(m.observer_coordinate)
```

Comment:
```suggestion
```

Reply:
Oh this is what you meant, sorry, I misunderstood